samples/server/petstore/php-flight/README.md

```diff
@@ -0,0 +1,10 @@
+## Requirements
+
+PHP 8.1 and later
+
+## Installation & Usage
+
+1. Set up flight as usual - see [Flight documentation](https://docs.flightphp.com/install)
+2. Generate using the OpenAPI generator
+3. Subclass some/all generated `Abstract*Api` and overwrite the methods you'd like handled. When implementing the `*Stream` methods, make sure to stream the response and not implement the non-stream method.
+4. Register routes for your subclassed apis: `RegisterRoutes::registerRoutes(new MyApiHandler());`
```

Comment:
So this is a good basic coverage on how to implement this, but after this gets merged into openapi, if you wouldn't mine touching base and we could put a more fleshed out example in the official documentation that people could use to implement a basic example, that would be awesome!

Reply:
sure - will do!